app/assets/stylesheets/custom.scss

```
@@ -1,5 +1,7 @@
 @import "bootstrap";
 
+$color-Gray: #777;
```

ko cần thêm color đằng trước đâu anh 
```suggestion
$gray: #777;
```